topics/contributing/tutorials/create-new-slides/tutorial.md

```diff
@@ -0,0 +1,358 @@
+        ![eu flag with the text: with the support of the erasmus programme of the european union](https://gallantries.github.io/assets/images/logosbeneficaireserasmusright_en.jpg)
+   ```
+
+   Funding bodies will be credited at the bottom of the tutorial with the appropriate funding statement, and will get a page in the hall of fame listing all tutorials that list them as a funder.
+
+   For an example of how this all looks, see the [R basics tutorial]({% link topics/data-science/tutorials/r-basics/tutorial.md %}) (top and bottom of the tutorial).
+
+{% assign kid_key = "Contributions Schema" %}
+{% assign kid_val = site.data['schema-tutorial']['mapping']['contributions'] %}
+{% include _includes/schema-render.html key=kid_key value=kid_val %}
+
+{% assign kid_key = "CONTRIBUTORS Schema" %}
+{% assign kid_val = site.data['schema-contributors'] %}
+{% include _includes/schema-render.html key=kid_key value=kid_val %}
+
+# Content
+
+The tutorial's content is written directly after the section of metadata. This is written in Markdown, a simple markup language.
+
+> <comment-title>Markdown</comment-title>
+>
+> Check [this cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to learn more how to use Markdown.
+{: .comment}
+
+The Markdown content is then transformed into a user friendly webpage through a templating system. With this approach, there is no need to add the name of every tutorial each time, since they are automatically added based on the tutorial's metadata.
+
+## Formatting
+
+Please see the [associated slide deck]({% link topics/contributing/tutorials/create-new-tutorial-slides/slides.html %}) for details on the formatting commands available. In general the markdown you already know will continue to work, but there are additional slide formatting commands available (e.g. `.pull-left[]` for classic two column layouts.)
+
+## Automated Video Recordings
+
+Please see the [dedicated tutorial on adding video support]({% link topics/contributing/tutorials/slides-with-video/tutorial.md %}) on adding automatic video recordings to your slides, if you're interested in that feature.
+
+## Adding images with captions
+
+To add an image in Markdown file, we need to use the markdown syntax for this: {% raw %}`!​[proper alt text describing the image for visually impaired learners](../../images/image.png)`{% endraw %}.
+
+We have also added a small plugin to handle captions for each image:
+
+![A textual description of the image](../../images/image_caption_screenshot.png "Example of an image with a caption ")<!-- Adding a space to the caption to not trigger figurigy skip_titles -->
+
+The prefix "Figure 1." is automatically added before its caption. This is done with the following Markdown syntax:
+
+{% raw %}
+```markdown
+!​[A textual description of the image](../images/image.png "Example of an image with a caption")
+```
+{% endraw %}
+
+### Guidelines on Alt vs Figcaption Text
```

Comment:
This section, the one before, and following ones look similar to the ones in "Tutorial" tutorial. Maybe we could put them in external Markdown and import them in both tutorials

Reply:
yeah it would be a good FAQ. I am often linking to it when reviewing tutorials